web/modules/custom/feedback/src/Controller/ReportController.php

```diff
@@ -1,141 +0,0 @@
-<?php
-
-/**
- * @file
- * Contains \Drupal\feedback\Controller\ReportController.
- */
-
-namespace Drupal\feedback\Controller;
-
-use Drupal\Core\Controller\ControllerBase;
-use Drupal\Core\Database\Connection;
-use Drupal\Core\Form\FormBuilderInterface;
-use Drupal\Core\Link;
-use Drupal\Core\Url;
-use Symfony\Component\DependencyInjection\ContainerInterface;
-use Symfony\Component\HttpFoundation\RequestStack;
-
-
-/**
- * Controller routines for report routes.
- */
-class ReportController extends ControllerBase {
-
-  /**
-   * The Database Connection.
-   *
-   * @var \Drupal\Core\Database\Connection
-   */
-  protected $database;
-
-  /**
-   * @var \Symfony\Component\HttpFoundation\RequestStack
-   */
-  protected $requestStack;
-
-  /**
-   * @var \Drupal\Core\Form\FormBuilderInterface
-   */
-  protected $formBuilder;
-
-  /**
-   * ReportController constructor.
-   *
-   * @param \Drupal\Core\Database\Connection $database
-   *
-   * @param \Symfony\Component\HttpFoundation\RequestStack $request_stack
-   *
-   * @param \Drupal\Core\Form\FormBuilderInterface $formBuilder
-   */
-  public function __construct(Connection $database, RequestStack $request_stack, FormBuilderInterface $formBuilder) {
-    $this->database = $database;
-    $this->requestStack = $request_stack;
-    $this->formBuilder = $formBuilder;
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public static function create(ContainerInterface $container) {
-    return new static(
-      $container->get('database'),
-      $container->get('request_stack'),
-      $container->get('form_builder')
-    );
-  }
-
-  /**
-   * A simple controller method to explain what the report is about.
-   */
-  public function description() {
-
-    $header = [
-      ['data' => $this->t('Feedback_id'), 'field' => 'f.feedback_id'],
-      ['data' => $this->t('User_ID'), 'field' => 'f.uid'],
-      ['data' => $this->t('Name'), 'field' => 'f.name'],
-      ['data' => $this->t('Email'), 'field' => 'f.email'],
-      ['data' => $this->t('Message')],
-      ['data' => $this->t('Created'), 'field' => 'f.created'],
-      ['data' => $this->t('Operation')],
-      ['data' => $this->t('Operation')],
-    ];
-
-    $getFilterValue = $this->requestStack->getCurrentRequest()->query->get('filter');
-
-    if (isset($getFilterValue)) {
-      $get_value = $this->database->escapeLike($getFilterValue);
-      $query = $this->database->select('feedback', 'f');
-      $query->fields('f');
-      $query->condition('f.name', $get_value  . '%', 'LIKE');
-
-    } else {
```

Comment:
install Drupal PHP code sniffer to reach drupal coding standards – https://www.drupal.org/docs/develop/standards/coding-standards